chain/indexer.go

```diff
@@ -744,6 +784,11 @@ type TipSetProcessor interface {
 	Close() error
 }
 
+type ConsensusProcessor interface {
```

Comment:
Could call it TipSet**s**Processor instead since it operates on a pair. I didn't use the TipSetProcessor since it would require API access in the task to fetch the parent tipset, and this implementation felt simpler. Do you think that's a fair tradeoff?